Sources/KeyboardManager/KeyboardManager.swift

```diff
@@ -137,7 +137,7 @@ open class KeyboardManager: NSObject, UIGestureRecognizerDelegate {
     /// - Parameter inputAccessoryView: The view to bind to the top of the keyboard but within its superview
     /// - Returns: Self
     @discardableResult
-    open func bind(inputAccessoryView: UIView) -> Self {
+    open func bind(inputAccessoryView: UIView, usingTabBar tabBar: UITabBar? = .none) -> Self {
```

Comment:
Why `.none` and not `nil` by default?

Reply:
they're synonyms. I can rename to `nil` if you'd prefer, but they mean the same thing. I often prefer .none since it reads better to me

Reply:
This change assumes someone is using a UITabBar. Would it be better if instead there was an offset property someone could just set? Or have this be a closure that returns an offset.

Reply:
great idea. I'll switch to a closure